dxp/demystifying_aui_input_tags.md

```diff
@@ -0,0 +1,727 @@
+| `id`                | Prepends the portlet namespace to the `id`, if no `id` is specified it's the same as the `name` attribute                                                                                                                                                                              |
+| `inlineField`       | A `boolean` value that renders the group inline. This adds the class `form-group-inline` to `form-group`. Use `inlineField="<%= true %>"` to render the group inline.                                                                                                                  |
+| `inlineLabel`       | This attribute accepts a `string` and adds the class `form-inline` to `<div class="form-group">` to display the `label` and `input` on one line. Use `inlineLabel="any text here"`.                                                                                                    |
+| `label`             | The text to use inside the `<label>` element, if no `label` is specified it's the same as the `name` attribute. Use `label=""` to prevent the taglib from outputting the `<label>` element. This attribute accepts language keys.                                                      |
+| `localized`         | See Localized section.                                                                                                                                                                                                                                                                 |
+| `name`              | Prepends the portlet namespace to the `name` value. If the name contains `--`, the generated name will be the substring following the `--`, excluding the last two characters and any prefixes. For example, the pattern `prefix--customName--` would result in the name `customName`. |
+| `localizeLabel`     | A `boolean` value that enables or disables localization in the `<label>` element (e.g., `localizeLabel="<%= false %>"`). This is `true` by default.                                                                                                                                    |
+| `onChange`          | Javascript to execute when the `change` event fires on the `input` (e.g., `onChange="console.log('onchange');"`.                                                                                                                                                                       |
+| `onClick`           | Javascript to execute when the `click` event fires on the `input` (e.g., `onChange="console.log('onclick');"`.                                                                                                                                                                         |
```

Comment:
"Javascript" → "JavaScript" on these two lines. Also:

![you-are-fired](https://user-images.githubusercontent.com/7074/95089657-ad248180-0724-11eb-9ed4-4bf2082eb512.gif)

Reply:
![party](https://user-images.githubusercontent.com/788266/95128989-1ef1d080-070f-11eb-92a0-587c4146e51f.gif)